### ISSUE TITLE
Wrong downloaded image file extension

### ISSUE THREAD

**Issue by dizcza** [facebookresearch/visdom#772]
**Reproduction Steps**
Taken from demo.py script:
```python
    viz.images(
        np.random.randn(20, 3, 64, 64),
        opts=dict(title='Random images', caption='How random.')
    )
```
Navigate to locahost:8097 "main" env, download the image by clicking on the download button. The downloaded file has a ".jpg" extension and cannot be opened - it must have a ".png" extension.

**Expected behavior**
"Random images.png" is downloaded.

**Additional context**
I thought the issue is in https://github.com/facebookresearch/visdom/blob/master/js/ImagePane.js#L87; I changed it to have a png suffix but it does not resolve the issue.

**Comment by kashyaprajharsh** [comment]
Sir, I want to work on this issue.

**Comment by dizcza** [comment]
If you're asking me, of course, I'll be glad if you or someone else will fix it. I just posted the issue here.

**Comment by rajdesai24** [comment]
<img width="1680" alt="Screenshot 2021-01-27 at 5 22 01 PM" src="https://user-images.githubusercontent.com/53701807/105987349-2d9c3300-60c4-11eb-8f4f-5837e01e0731.png">

**Comment by rajdesai24** [comment]
<img width="871" alt="Screenshot 2021-01-27 at 5 23 16 PM" src="https://user-images.githubusercontent.com/53701807/105987485-5a504a80-60c4-11eb-97b2-24e6b10f7a8f.png">

**Comment by rajdesai24** [comment]
This says it downloads jpg when we specify it specifically. Kindly check if you've done that, it could be the reason you might be facing the issue.

**Comment by dizcza** [comment]
@rajdesai24 
please read the issue description once again.
Do I define a jpgquality in the test script? - No. ---> "If not defined, the image will be saved as a PNG" - this is the expected behavior.
But instead, I get an image with a JPG extension that cannot be opened by any JPG readers because it is NOT a JPG.

**Comment by rajdesai24** [comment]
Yes, understood. Working on it @dizcza, thank you for clearing it out.

**Comment by Shrot101** [comment]
> I attempted to reproduce using the latest master branch following the provided steps, and the image downloads correctly as a .jpg file and opens without issue. Could you confirm if this problem still occurs on the latest version? [@dizcza](https://github.com/dizcza) [@hpdang](https://github.com/hpdang) [@mariobehling](https://github.com/mariobehling)

Hello, please confirm.

**Comment by dizcza** [comment]
The issue was opened more than 5 years ago. It's no longer relevant to my work.
